src/main/java/com/plugin/frontend/actions/FormatAreaAction.java

```diff
@@ -15,50 +15,40 @@
 
 import java.io.*;
 import java.util.ArrayList;
+import java.util.Objects;
 
 public class FormatAreaAction extends AnAction {
+
     @Override
     public void actionPerformed(@NotNull AnActionEvent e) {
         JavaFormatter test = new JavaFormatter();
-        VirtualFile file = FileDocumentManager.getInstance().getFile(e.getData(LangDataKeys.EDITOR).getDocument());
+        VirtualFile file = FileDocumentManager.getInstance().getFile(Objects.requireNonNull(e.getData(LangDataKeys.EDITOR)).getDocument());
 
         Editor editor = e.getData(PlatformDataKeys.EDITOR);
 
-        int startOffset = editor.getSelectionModel().getSelectionStart();
-        int endOffset = editor.getSelectionModel().getSelectionEnd();
+        assert editor != null;
```

Comment:
It's bad to use assert. You should create try-catch block and alert user about this exception. Or u should create simple condition and alert user.